src/styles/blocks/card/blockscard/starReviewsPrice.scss

```diff
@@ -0,0 +1,77 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+
+  margin-top: 8px;
+  margin-bottom: 8px;
+  column-gap: 17px;
```

Comment:
You are using `column-gap` property here. This property is used with grid layout. If you want to use gap between flex items, you should use `gap` property. Please, review your layout and use the appropriate properties.